.lintstagedrc

```diff
@@ -0,0 +1,4 @@
+{
+  "*.tsx": ["prettier --write"],
```

Comment:
I can't test it now, but ensure `ts` files also format.

Reply:
fixed